apps/src/lab2/levelEditors/aiCustomizations/ModelCardFields.tsx

```diff
@@ -0,0 +1,62 @@
+import {ModelCardInfo} from '@cdo/apps/aichat/types';
+import React, {useContext} from 'react';
+import MultiItemInput from './MultiItemInput';
+import moduleStyles from './edit-ai-customizations.module.scss';
+import {MAX_ASK_ABOUT_TOPICS} from '@cdo/apps/aichat/constants';
+import {UpdateContext} from './UpdateContext';
+
+const modelCardFieldsAndLabels: [keyof ModelCardInfo, string][] = [
+  ['description', 'Description'],
+  ['intendedUse', 'Intended Use'],
+  ['limitationsAndWarnings', 'Limitations and Warnings'],
+  ['testingAndEvaluation', 'Testing and Evaluation'],
+];
+
+const ModelCardFields: React.FunctionComponent = () => {
+  const {setModelCardPropertyValue, aiCustomizations} =
+    useContext(UpdateContext);
+  const askAboutTopics = aiCustomizations.modelCardInfo?.value?.askAboutTopics;
+  return (
+    <div className={moduleStyles['model-card-fields']}>
+      {modelCardFieldsAndLabels.map(([property, label]) => {
+        return (
+          <div key={property}>
+            <label htmlFor={`modelCard-${property}`}>{label}</label>
+            <textarea
+              id={`modelCard-${property}`}
+              value={aiCustomizations.modelCardInfo?.value?.[property] || ''}
+              onChange={e =>
+                setModelCardPropertyValue(property, e.target.value)
+              }
+              className={moduleStyles.textarea}
+            />
+          </div>
+        );
+      })}
+      <label>Ask About Topics</label>
```

Comment:
In some recent mockups we sent to Mark, Sam and I settled on these being called "Example Prompts and Topics" for the model card. Even though the design isn't settled, that phrasing feels set - can we update this label to match that?

Reply:
I didn't see see this section in the mocks I was working off of on the student side...maybe the Figma I was using is out of date? [This is what I was using](https://www.figma.com/file/yGY2tcdEwvS3Xnox4eP4BO/Mockups-Discussion-2%2F21?type=whiteboard&node-id=0-1&t=R2YN8xazIKhcsPKr-0)